<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>구글 한줄 변역기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>한국어 입력</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_kor_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>461</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:2px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trans_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>궁서체</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:2px;
background-color: rgb(255, 255, 127);</string>
    </property>
    <property name="text">
     <string>번 역</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>일본어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중국어</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_eng_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>550</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:2px;
border-color:red;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_jap_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>550</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:2px;
border-color: rgb(0, 0, 255);
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_cha_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>550</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:2px;
border-color: rgb(255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>510</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼편지체</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
